test/fn.js

```diff
@@ -7,10 +7,3 @@ test('function', function (t) {
     t.equal(inspect(obj), '[ 1, 2, [Function: f], 4 ]');
 });
 
-test('function name', function (t) {
```

Comment:
looks like this test fails in master on node 6+ due to function name inference.

I'll fix this in master rather than just removing the test here.